airbyte-integrations/connectors/source-jira/source_jira/manifest.yaml

```diff
@@ -0,0 +1,1333 @@
+version: 0.72.1
+type: DeclarativeSource
+
+definitions:
+  # Authenticators
+
+  authenticator:
+    type: BasicHttpAuthenticator
+    username: "{{ config['email'] }}"
+    password: "{{ config['api_token'] }}"
+
+  # Error handler
+  error_handler:
+    type: DefaultErrorHandler
+    max_retries: 10
+    response_filters:
+      - http_codes: [400]
+        action: IGNORE
+
+  # Requesters
+
+  requester:
+    type: HttpRequester
+    http_method: GET
+    url_base: "https://{{ config['domain'] }}/rest/api/3/"
+    authenticator: "#/definitions/authenticator"
+    error_handler: "#/definitions/error_handler"
+
+  requester_v1:
```

Comment:
I'm curious why we're using multiple API versions. Are we in the process of migrating everything to v3?

Reply:
I'm not entirely sure why, but I suspect it's because the latest version doesn't include an API for retrieving sprint issues, for example.